### ISSUE TITLE
Building source dist with python3 fails and should fail

### ISSUE THREAD

**Issue by benjaoming** [learningequality/kolibri#3319]
### Observed behavior

Our current build pipeline is intended to be run with Python 2.7 because some of the requirements are only available when invoking pip as such.

I will create an assertion that fails with a proper message shortly....

CC: @lyw07 

### Errors and logs

```
rm -f kolibri/dist/kolibri_exercise_perseus_plugin/static/mathjax/kathjax.py
python build_tools/py2only.py # move `future` and `futures` packages to `kolibri/dist/py2only`
Traceback (most recent call last):
  File "build_tools/py2only.py", line 46, in <module>
    hide_py2_modules()
  File "build_tools/py2only.py", line 28, in hide_py2_modules
    _move_modules_to_py2only(module)
  File "build_tools/py2only.py", line 34, in _move_modules_to_py2only
    shutil.move(module_src_path, module_dst_path)
  File "/usr/lib/python2.7/shutil.py", line 302, in move
    copy2(src, real_dst)
  File "/usr/lib/python2.7/shutil.py", line 130, in copy2
    copyfile(src, dst)
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: 'kolibri/kolibri/dist/configparser'
Makefile:125: recipe for target 'staticdeps' failed
make: *** [staticdeps] Error 1
```

**Comment by lyw07** [comment]
I remember this `IOError: [Errno 2] No such file or directory: 'kolibri/kolibri/dist/configparser'` error happened last time too? 
I checked the modules I try to pull from `future`: https://github.com/PythonCharmers/python-future/blob/003763d92a0528e6a56c5132c7821b4de413bc83/src/future/standard_library/__init__.py#L792, but `configparser` is not there...

**Comment by benjaoming** [comment]
Fixed in #3321
